CHANGELOG.md

```diff
@@ -4,6 +4,20 @@ See the [change log guidelines](http://keepachangelog.com/) for information on h
 
 ## [Unreleased]
 
+## [5.0.1]
+
+### Added
+- Support to WFS reader using the original data from download
```

Comment:
I would either remove it or rephrase it.
It is something that is currently not configurable via the UI and is a very niche use case.
With the new setting the WFS reader also downloads the source as files, it did also before use the original data for loading instances.